vendor/validator.js

```diff
@@ -6,63 +6,63 @@
  * https://github.com/1000hz/bootstrap-validator
  */
 
-+function ($) {
++(function($) {
```

Comment:
Because this is a third-party vendor, I think the best way is to ignore it from eslint.